src/signals/incident/components/form/MapSelectors/Asset/AssetList/hooks/useSelectionProps.tsx

```diff
@@ -33,15 +33,16 @@ export const useSelectionProps = ({
   const { setItem } = useContext(AssetSelectContext)
   const { maxAssetWarning } = useSelector(makeSelectMaxAssetWarning)
   const featureStatusType = featureStatusTypes.find(
-    ({ typeValue }) => typeValue === status
+    ({ typeValue }) => typeValue === feature.status
   )
   const item: Item = {
     ...feature,
     address: undefined,
     status: featureStatusType?.typeValue,
   }
 
-  if (selection?.find((item) => item.id === feature.id)) return null
+  if (selection?.find((item) => item.id?.toString() === feature.id.toString()))
```

Comment:
it can be undefined, string or number. It would be a good idea to figure this all out and align the types accordingly. But I think it's out of scope for this PR and still you would have the uncertain types of other municipalities.